### PR TITLE
apply recent dapp-offer-up dependency work

### PR DESCRIPTION
**Pull request by dckc**
fixes:

```
not ok 7 - Error: Cannot find module '/Users/tgreco/dapp-airdrop/contract/node_modules/@noble/hashes/pbkdf2.js'
```

context:
 - https://github.com/Agoric/agoric-sdk/discussions/9958
 - https://github.com/Agoric/agoric-sdk/pull/10034
 - https://github.com/hindley-milner-systems/dapp-ertp-airdrop/issues/57#issuecomment-2339312725

background:
 - https://github.com/Agoric/agoric-sdk/pull/9576
 - https://github.com/Agoric/agoric-sdk/pull/10034

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -59,5 +59,5 @@
     "eslint-plugin-ui-testing": "^2.0.1",
     "serve": "^14.2.1"
   },
-  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
-}
\ No newline at end of file
+}
+  "packageManager": "yarn@4.4.1"
```
Comment (dckc) [inline package.json:64]: oops. bad merge.

Comment (dckc) [reply package.json:64]: ah. I must have fixed it later.

Comment (tgrecojs) [reply package.json:64]: this build was still failing 🤔 

I managed to resolve it in #105! 🥳

**Comment by tgrecojs** [comment]
closing this as it's been fixed and merged into dev.

**the fix**: update the top-level package.json so that "resolutions" contains  `"@noble/hashes": "1.5.0"`. 

from what i understand, this change will inform the package manager to reference a particular version of a library whenever it is referenced, including if it is referenced in an external dependency. In our code, #105 instructs the package manager to reference v1.5.0 when cosmjs requests it. 

s/o @jovanni for the tip

**Pull request closed without merging**